src/sentry/integrations/slack/client.py

```diff
@@ -137,5 +137,7 @@ def request(
             **kwargs,
         )
         if not raw_response and not response.json.get("ok"):
+            if kwargs.get("log_response_with_error", False):
+                self.logger.info("rule.fail.slack_post.log_response", extra={"response": response})
```

Comment:
the response is a django class object no? Won't this just log the memory address and not the contents? Are we looking for `response.__dict__`?